include/solvers/multiphysics_interface.h

```diff
@@ -819,6 +914,9 @@ class MultiphysicsInterface
   // Data structure to store all physics which were enabled
   std::vector<PhysicsID> active_physics;
 
+  // Fluid solver is a constraing solver. (GLS_NITSCHE ,GLS_SHARP)
+
+
```

Comment:
What? contraing? like constrained?

Reply:
This has been removes it was use in an other architecture that wasn't working

Reply:
same has above